app/assets/javascripts/course.ts

```diff
@@ -0,0 +1,378 @@
+import { initDragAndDrop } from "./drag_and_drop.js";
+import { fetch, getURLParameter } from "./util.js";
+import { ScrollSpy } from "./scrollspy";
+import { searchQuery } from "./search";
+
+function loadUsers(_baseUrl: string, _status): void {
+    const status = _status || getURLParameter("status");
+    searchQuery.queryParams.updateParam("status", status);
+}
+
+function initCourseMembers(): void {
+    function init(): void {
+        initUserTabs();
+        initLabelsEditModal();
+    }
+
+    function initUserTabs(): void {
+        const userTabs = document.getElementById("user-tabs");
+        if (userTabs !== null) {
+            const baseUrl = userTabs.getAttribute("data-baseurl");
```

Comment:
```suggestion
            const baseUrl = userTabs.dataset.baseurl;
```
A similar change can be done for other data attributes.